create-ava-rule.js

```diff
@@ -65,12 +188,12 @@ module.exports = () => {
 	/* eslint quote-props: [2, "as-needed"] */
 	const predefinedRules = {
 		ImportDeclaration: node => {
-			if (!isTestFile && deepStrictEqual(espurify(node), avaImportDeclarationAst)) {
+			if (!isTestFile && avaImportDeclarationAsts.some(ast => deepStrictEqual(espurify(node), ast))) {
 				isTestFile = true;
 			}
 		},
 		VariableDeclarator: node => {
-			if (!isTestFile && deepStrictEqual(espurify(node), avaVariableDeclaratorAst)) {
+			if (!isTestFile && avaVariableDeclaratorAsts.some(ast => console.log(JSON.stringify(espurify(node), null, 2)) || deepStrictEqual(espurify(node), ast))) {
```

Comment:
Did you mean to commit the `console.log()` call?

Reply:
Oops 🤦‍♂️ fixed